src/pylibsshext/session.pyx

```diff
@@ -334,6 +327,7 @@ cdef class Session(object):
         if rc != libssh.SSH_OK:
             return
 
+        # FIXME The SHA1 hashes are non-standard -- the most common these days are SHA256 hashes for fingerprints
```

Comment:
We should probably move this FIXME into an issue on the tracker.

Reply:
Created #701. Will remove from this PR as it is clearly unrelated. I probably just needed to write it somewhere.